README.md

```diff
@@ -18,3 +18,27 @@ The following assumes the use of `node@>=10`.
 ## Run Tests
 
 `yarn test`
+
+## Generate Merkle Tree
+
+`npx ts-node  scripts/generate-merkle-root.ts -i "input-json-file"`
+
+
+## Deploy Merkle distributor
+
+First, set-up the configuration variables using Harhad:
+
+`npx hardhat vars set XXX_PRIVATE_KEY`
+
+`npx hardhat vars set XXX_RPC_URL`
+
+`npx hardhat vars set TENDERLY_CHAIN_ID`
```

Comment:
```suggestion
`npx hardhat vars set XXX_CHAIN_ID`
```

Reply:
This is intentional...because for mainnet it is not used (it is always 1) but we can add it if you prefer to be explicit.

Reply:
Maybe then add something like: "If using Tenderly Virtual Testnets, provide TENDERLY_CHAIN_ID".